resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -507,6 +512,12 @@ protected void doResolve(String inetHost,
             return;
         }
 
+        if (inetHost.isEmpty()) {
```

Comment:
Not to `SimpleNameResolver` but to `InetNameResolver`? `SimpleNameResolver` can be used for non-INET scenarios.